### DISCUSSION TITLE
#[Url] and the search parameters

### DISCUSSION THREAD

**Discussion by musiccollect** [livewire/livewire#8228] [1. Help] [answered]
I was expecting the text someone typed in a table's search box to start appearing as ?search=the+text at the end of the URL immediatey they start typing. But it only seems to happen after the results are returned and shown in the table. I can manually add the search string to the URL and the table results appear straight away. 

I've tried numerous combinations of #[Url()] in the component file and wire:model.live="search" in the blade file.

**Comment by PhiloNL** [comment, marked as answer]
Hi @musiccollect,
The `Url` attribute seems to work (type something in the input field):
https://xpjkq.wirebox.app:8443/

As you can see, the URL is added and updated while typing.

The code: https://wirebox.app/b/xpjkq

**Comment by musiccollect** [comment]
Thanks for the suggestion. Actually I discovered that wire:model.live.blur gave me what I needed.
